.github/workflows/yjit-ubuntu.yml

```
@@ -47,7 +47,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - test_task: "check-yjit-bindings"
+          - hint: "To fix: use patch in logs" # this shows up first in the web interface
+            test_task: "check-yjit-bindings"
```

```suggestion
          - test_task: "check-yjit-bindings"
             hint: "To fix: use patch in logs" # this shows up first in the web interface
```
Can we keep the make task name coming first? We try to make job names look like a make command `make check-yjit-bindings`, but now it looks like `make To fix: use patch in logs check-yjit-bindings`. If we want to make it a bit easier to see, maybe we could say `test_task: "yjit-bindgen"` instead.